metomi/rose/scripts/rose-tutorial

```diff
@@ -28,9 +28,16 @@
 #
 # DESCRIPTION
 #     Make a copy of one of the tutorial SUITE in the cylc-run directory.
-#
 #-------------------------------------------------------------------------------
 set -eu
+
+if [[ $1 == '--help' ]]; then
+    # shellcheck source=metomi/rose/etc/lib/bash/rose_usage
+    . "$(rose resource lib/bash/rose_usage)"
+    rose_help
+    exit 0
+fi
+
 mkdir -p "$HOME/cylc-run"
 
 usage () {
```

Comment:
This doesn't work for me - at Rose 1 line 47 was   
```bash
ls -1 "${ROSE_HOME}/etc/tutorial/" | sed -e 's/^/    /'
```

I'm not sure how this find command is supposed to produce the desired result. Going digging into the change history.